iphone/Maps/UI/Help/AboutController/AboutController.swift

```diff
@@ -491,15 +505,10 @@ private extension AboutController {
                                  openOutlook(subject: subject, body: body, recipients: toRecipients))) {
       return
     }
+    
     // From iOS 14, it is possible to change the default mail app, and mailto should open a default mail app.
-    if MWMMailViewController.canSendMail() {
-      let vc = MWMMailViewController()
-      vc.mailComposeDelegate = self
-      vc.setSubject(subject)
-      vc.setMessageBody(body, isHTML:false)
-      vc.setToRecipients(toRecipients)
-      vc.navigationBar.tintColor = UIColor.whitePrimaryText()
-      self.present(vc, animated: true, completion:nil)
+    if openDefaultApp(subject: subject, body: body, recipients: toRecipients){
```

Comment:
```suggestion
    if openDefaultMailApp(subject: subject, body: body, recipients: toRecipients) {
```

Reply:
are you updating the name of the function on line 485 as well? I see only 510 being updated, and it would not work if we just change this line.

Reply:
As a reviewer, I'm suggesting to use a more clear name of the function. It's an author's responsibility to make it work (or disagree with arguments).

Reply:
Sorry, i do agree with the changes. I was thinking to sign and accept the changes from here to simplify. In any case i did update the naming